CRM/Contribute/BAO/Contribution.php

```diff
@@ -5253,10 +5251,11 @@ public static function updateMembershipBasedOnCompletionOfContribution($contribu
         // Passing num_terms to the api triggers date calculations, but for pending memberships these may be already calculated.
         // sigh - they should  be  consistent but removing the end date check causes test failures & maybe UI too?
         // The api assumes num_terms is a special sauce for 'is_renewal' so we need to not pass it when updating a pending to completed.
+        // ... except testCompleteTransactionMembershipPriceSetTwoTerms his this line to the above is obviously not true....
```

Comment:
@eileenmcnaughton This comment doesn't quite make sense.. I think there are a couple of typos?

Reply:
@mattwire is that better?